packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/Balance/SfoxPendingBalance/selectors.js

```diff
@@ -0,0 +1,22 @@
+import { selectors } from 'data'
+import { add, compose, filter, reduce, pathOr } from 'ramda'
+import { Exchange } from 'blockchain-wallet-v4/src'
+
+const isBuyProcessing = trade => trade.isBuy && trade.state === 'processing'
+
+const extractPendingBalance = trades => {
+  const pendingBuyTrades = compose(filter(isBuyProcessing))(trades)
```

Comment:
compose is unneeded

Reply:
Removing, was composing isBuy and isProcessing until I made them one